### PR TITLE
Only get number of followers if actually needs to.

### PR DESCRIPTION
**Pull request by pekapa**
Avoid going to the image poster profile page and back to the image if we are not filtering the likes by number of followers.

This is not about the logged in user (the one running the script). This is about the owner of the picture being liked.

### CODE REVIEW

**Comment by eduardocalazansjr** [comment]
The profile page needs to be opened for two reasons:
 - For comparison and write on FolowerNum.txt (it's kinda of a log/performance metter)
 - For future uses of the _userData, that only resides on the user profile page.

**Comment by pekapa** [comment]
@vlntdds, this is the poster user profile, from the image being liked.
Not sure if this was clear enough but this is not the logged user profile.

**Comment by eduardocalazansjr** [comment]
Oh, sorry, my bad.
Could you please explain better on the PR?

Also, take a look on https://github.com/timgrossmann/InstaPy/pull/109#issuecomment-294316590

**Comment by pekapa** [comment]
Yeah, I fixed that in this commit as well. Used string concat as the previous prints to keep it consistent.

**Comment by timgrossmann** [comment]
@pekapa Thank you, I'll edit the position of the follower get just a tiny bit

File: instapy/like_util.py [hunk 1]
```diff
@@ -65,24 +65,24 @@ def check_link(browser, link, dont_like, ignore_if_contains, username, like_by_f
...
-    return True, user_name, 'Number of followers does not reach limit'
+  """Find the number of followes the user has"""
+  if like_by_followers_upper_limit or like_by_followers_lower_limit:
+    userlink = 'https://www.instagram.com/' + user_name
+    browser.get(userlink)
+    sleep(1)
+    num_followers = browser.execute_script("return window._sharedData.entry_data.ProfilePage[0].user.followed_by.count")
+    browser.get(link)
+    sleep(1)
+    print('Number of Followers: ' + num_followers)
```
Comment (littleswany) [inline instapy/like_util.py:80]: @pekapa This causes an issue when printing the follower number:

`  File "./instapy/like_util.py", line 100, in check_link
    print('Number of Followers: ' + num_followers)
TypeError: cannot concatenate 'str' and 'int' objects`

I will submit a PR to undo this and fix the issue unless there is another reason?

Comment (timgrossmann) [reply instapy/like_util.py:80]: should be fixed by #122 @pekapa

**Pull request merged**